### ISSUE TITLE
The main inserter has a visually hidden 'Close' button that shows only its tooltip on focus

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#45974]
### Description

Noticed while debugging https://github.com/WordPress/gutenberg/issues/45973

When tabbing through the elements within the main block Inserter, at some point focus goes to a visually hidden 'Close block inserter' button. The button is visually hidden but it still shows its tooltip on focus:

<img width="426" alt="Screenshot 2022-11-22 at 15 57 24" src="https://user-images.githubusercontent.com/1682452/203354522-d619185d-ba9a-44d3-8954-1f23c44c3d8c.png">

Ideally, for better accessibility, interactive controls should be visible. I do understand why this button is visually hidden but this design pattern is also inconsistent with other panels rendered within the secondary sidebar. For example, the Document overview panel (formerly 'List view') does have a visible close button:

<img width="401" alt="Screenshot 2022-11-22 at 16 34 20" src="https://user-images.githubusercontent.com/1682452/203356191-631cfe88-cc9e-43fd-b9f8-5491d37dbec9.png">

Ideally, this button should be visible.
Alternatively, it should not show a tooltip that visually looks 'detached' from the UI. 

### Step-by-step reproduction instructions

- Edit a post.
- Make sure the editor is in 'Select' mode. This is necessary until https://github.com/WordPress/gutenberg/issues/45973 gets solved.
- Open the block inserter and tab through its focusable elements by using the Tab key.
- Observe that at some point a 'Close block inserter' tooltip appears.
- Can be reproduced in the site editor and widgets editor as well.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by javierarce** [comment]
I can imagine that the lack of a close button is related to the [original behaviour](https://github.com/WordPress/gutenberg/issues/26223) of this sidebar. 

If we added a close button, I guess we should also remove the option to autoclose the sidebar by clicking outside (to make it consistent with the other columns). 

I'm more inclined to keep the current behaviour and remove the element that is triggering the tooltip.

**Comment by afercia** [comment]
Note: this button becomes visible on small screens:

<img width="860" alt="Screenshot 2022-11-30 at 10 59 15" src="https://user-images.githubusercontent.com/1682452/204769745-e30c6e15-5866-494f-99e1-0b1fae601f95.png">

**Comment by javierarce** [comment]
Oh, I hadn't noticed that, but that makes total sense. Otherwise it wouldn't be possible to close the sidebar on mobile. 

Then, if we decide to go with the idea of not showing the close button, we'll need to hide it with CSS instead of removing it.

**Comment by afercia** [comment]
Closing as https://github.com/WordPress/gutenberg/pull/61421 made the Close button always visible.
